inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -143,10 +144,15 @@ def _get_type(value):
             'BAI': 'INSPIRE BAI',
             'CERN': 'CERN',
             'DESY': 'DESY',
+            'GOOGLESCHOLAR': 'GOOGLESCHOLAR',
             'INSPIRE': 'INSPIRE ID',
             'KAKEN': 'KAKEN',
             'ORCID': 'ORCID',
+            'RESEARCHID': 'RESEARCHERID',
+            'RESEARCHERID': 'RESEARCHERID',
```

Comment:
Nope, this is working around a misspelling in the data: https://github.com/inspirehep/inspire/issues/211#issue-172317903. Actually I should add the ones about `ORCID`.